src/clarity-angular/datagrid/_datagrid.clarity.scss

```diff
@@ -542,6 +543,11 @@
         }
     }
 
+    .datagrid-foot-description {
```

Comment:
Why are we going from `footer` to `foot`? This seems inconsistent with other components where we use `footer`...